### PR TITLE
Move talk global settings to settings dialog

### PR DESCRIPTION
**Pull request by ma12-co**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Comment by nickvergessen** [comment]
Style/UI related PR without screenshot comparison :fearful:

**Comment by nickvergessen** [comment]
Can you base on #4027
I handled the tabs there already

**Comment by ma12-co** [comment]
> Style/UI related PR without screenshot comparison fearful

Screenshot from 2020-08-20 19-05-20

File: src/components/SettingsDialog/SettingsDialog.vue [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+		}
+	},
+
+	computed: {
+		attachmentFolder() {
+			return this.$store.getters.getAttachmentFolder()
+		},
+
+		defaultLocationHint() {
+			return t('spreed', `Choose in which folder talk attachments should be saved`)
```
Comment (nickvergessen) [inline src/components/SettingsDialog/SettingsDialog.vue:87]:
```suggestion
			return t('spreed', 'Choose in which folder shared files should be saved')
```

File: src/components/SettingsDialog/SettingsDialog.vue [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+	mounted() {
+		EventBus.$on('show-settings', this.handleShowSettings)
+		this.attachmentFolderLoading = false
+	},
+
+	methods: {
+
+		selectAttachmentFolder() {
+			const picker = getFilePickerBuilder(t('spreed', 'Select default location for attachments'))
```
Comment (nickvergessen) [inline src/components/SettingsDialog/SettingsDialog.vue:106]:
```suggestion
			const picker = getFilePickerBuilder(t('spreed', 'Select location for attachments'))
```

**Pull request merged**